Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj

```diff
@@ -1212,7 +1223,7 @@
 						ProvisioningStyle = Automatic;
 					};
 					6700165F1940F51400A9E103 = {
-						DevelopmentTeam = R63EM248DP;
```

Comment:
We should probably back this out, at least manually (with vi or whatever). I can do that if it's tough otherwise.

Reply:
Right, could you please do that?
I had to use our team and even when reverting outside xcode seems easy, I wouldn't like to mess with it.

Reply:
Ernest or I can revert once this is merged, since this is from your fork. You can also use any text editor to modify the field in the file. You should still be able to build and deploy to a simulator. You can also perhaps go to a previous commit from before this change, copy out `Branch-TestBed.xcodeproj/project.pbxproj` to `/tmp` or anywhere, then copy it back in after reverting to this `HEAD` revision, depending if there were other changes to the file in the same commit. Not a big deal. There are numerous ways to deal with it now or later.